tests/fixture/tmpnet/node_process.go

```diff
@@ -114,11 +114,8 @@ func (p *NodeProcess) Start(w io.Writer) error {
 
 	// All arguments are provided in the flags file
 	cmd := exec.Command(p.node.RuntimeConfig.AvalancheGoPath, "--config-file", p.node.getFlagsPath()) // #nosec G204
-
 	// Ensure process is detached from the parent process so that an error in the parent will not affect the child
-	cmd.SysProcAttr = &syscall.SysProcAttr{
-		Setsid: true,
-	}
+	configureDetachedProcess(cmd)
```

Comment:
To clarify - this PR is to make it so that this code can _compile_ on windows, but there is no expectation for it to _run_ on windows?